models/klaviyo__flows.sql

```diff
@@ -0,0 +1,37 @@
+with flow as (
+
+    select *
+    from {{ var('flow') }}
+),
+
+flow_metrics as (
+
+    select *
+    from {{ ref('int_klaviyo__campaign_flow_metrics') }}
+    where last_touch_flow_id is not null -- only pull flows
```

Comment:
I don't think this is necessary bc the join logic should automatically filter out any records that don't have a flow_id